wled00/data/settings_sync.htm

```diff
@@ -127,7 +127,7 @@ <h3>WLED Broadcast</h3>
 Send Alexa notifications: <input type="checkbox" name="SA"><br>
 Send Philips Hue change notifications: <input type="checkbox" name="SH"><br>
 Send Macro notifications: <input type="checkbox" name="SM"><br>
-Send notifications twice: <input type="checkbox" name="S2"><br>
+Number of UDP Messages: <input name="UR" type="number" min="1" max="255" class="d5" required><br><br>
```

Comment:
I would change the description to something like "Retransmissions:" or "UDP packet retransmissions:"